### PR TITLE
Add 3gppp and fortinet VSA to directory

### PR DESCRIPTION
**Pull request by mnowa**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.tinyradius</groupId>
 	<artifactId>tinyradius</artifactId>
-	<version>1.1.0</version>
+	<version>1.1.1-SNAPSHOT</version>
```
Comment (ctran) [inline pom.xml]: Why is this needed?

Comment (mnowa) [reply pom.xml]: I changed the dictionary therefore I also changed the master version (from 1.1.0 to 1.1.1-SNAPSHOT).  This is standard practice in maven projects that we do not change already released version (e.g 1.1.0). Instead we first create a new SNAPSHOT version (e.g. 1.1.1-SNAPSHOT),  applied one or more commits to that version and finally someone (e.g. project leader) releases new non SNAPSHOT version (e.g. 1.1.1) of the artifact.
